apps/sms/test/marionette/mocks/mock_message_thread.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
```

Comment:
Injects: At first, I thought only `mock_test_storage.js` was needed for generating a thread, and the test didn't run. I thought abstracting it behind a function would make the intent clearer: If you want to generate a thread, you need these and only these 2. Do you think I should remove the abstraction?

Thread generation: You're right, I missed that the same algorithm was used in `conversion_banner_test.js`. I'll extract this function too.

`uniqueIdCounter`: I see. Let's create a static variable in the mock class to store the next ID.